index.html

```diff
@@ -3069,6 +3100,11 @@ <h2>Integrity of Related Resources</h2>
 an object in the <code>relatedResource</code> array.
         </p>
         <p>
+Any failure to verify content integrity information that is vital to the
+validity of a <a>conforming document</a>, such as the integrity of content
+identified by related `@context` URLs, SHOULD result in a validation error.
```

Comment:
This seems like it should be a MUST, but that MUST should appear in the place that uses the related resource, rather than here. The wording here could be something like 

```
The authors of any specification that fetches a resource based on the `id`
of an object inside a <a>conforming document</a> need to consider whether
that resource's content is vital to the validity of that document. If it is,
the specification MUST produce a validation error unless the resource has
the expected media type and its bytes hash to the expected digest.
```